kore/src/Kore/AST/ApplicativeKore.hs

```diff
@@ -0,0 +1,61 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+module Kore.AST.ApplicativeKore
+    ( completeDefinition ) where
+
+import Control.Comonad
+import Data.Foldable
+import Kore.AST.Common
+import Kore.AST.MetaOrObject
+import Kore.AST.Sentence
+import Kore.AST.Valid
+import Kore.Step.Pattern hiding
+       ( freeVariables )
+
+completeDefinition :: VerifiedPureDefinition Object -> VerifiedPureDefinition Object
+completeDefinition Definition { definitionAttributes, definitionModules } =
+    Definition
+    { definitionAttributes
+    , definitionModules = map completeModule definitionModules
+    }
+
+completeModule :: VerifiedPureModule Object -> VerifiedPureModule Object
+completeModule Module { moduleName, moduleSentences, moduleAttributes } =
+    Module
+    { moduleName
+    , moduleSentences = concat (map completeSentence moduleSentences)
+    , moduleAttributes
+    }
+
+completeSentence :: VerifiedPureSentence Object -> [VerifiedPureSentence Object]
+completeSentence ( SentenceAxiomSentence
+                     SentenceAxiom
```

Comment:
This file and some of the code in the other files do not follow the indentation rules as described in
https://github.com/kframework/kore/wiki/Coding-Standards
(e.g. indentation levels divisible by 4)